src/users.js

```diff
@@ -0,0 +1,9 @@
+const getRandomUser = (userList) => {
+  if(!userList) {
+    return `User not found`;
+  }
+  const indexPosition = Math.floor(Math.random() * userList.length);
+  return userList[indexPosition];
+};
+
+export {getRandomUser}
```

Comment:
This is doing the same thing as the getRandomRecipe function, we should combine them and change the names of the parameters so that it makes more sense